index.html

```diff
@@ -0,0 +1,877 @@
+                    <li>Wednesday - October 28, 2020</li>
+                </ul>
+                <ul class="cards-list">
+                    <li class="cards-article">APPS</li>
+                    <li class="cards-title">Praise Hephaestus! Microsoft Office IPad apps now have trackpad support</li>
+                    <li>
+                        <ul class="data">
+                            <li>by callum booth</li>
+                            <li><img class="cards-icon" src="./assets/images/clock-128.png" alt=""><span>1 hour
+                                    ago</span> </li>
+                            <li><img class="cards-icon" src="./assets/images/sharethis-3-128.png" alt="">3</li>
+                        </ul>
+                    </li>
+                </ul>
+            </a>
+            <a href="" class="cards main-2">
+                <ul class="cards-list">
+                    <li class="cards-title">Watch all TNW2020 talks on-demand</li>
+                </ul>
+            </a>
+            <a href="" class="cards main-3">
+                <ul class="cards-list">
+                    <li class="cards-article">WORLD</li>
+                    <li class="cards-title">NASA discovers the moon is wet - now scientists want to go back</li>
+                    <li>
+                        <ul class="data">
+                            <li>by The Conversation</li>
+                            <li><img class="cards-icon" src="./assets/images/clock-128.png" alt=""><span>1 hour
+                                    ago</span> </li>
+                            <li><img class="cards-icon" src="./assets/images/sharethis-3-128.png" alt="">5</li>
+                        </ul>
+                    </li>
+                </ul>
+            </a>
+        </section>
+        <div class="main-flex">
+            <section class="latestnews">
+                <div><a href="#" class="orange-headings main-headings">
+                        Latest News >
+                    </a>
+                </div>
+                <div class="cards-flex">
+                    <div class="cards">
+                        <a class="img1" href="#"></a>
+                        <!-- <img src="./assets/images/uber-amsterdam-british-drivers-employee-gdpr-data-car-court-.jpg"> -->
+                        <div><a href="#" class="title">Uber is being sued by drivers claiming to have been sacked by its
+                                algorithm </a>
+                            <ul>
+                                <li><a href="#" class="credits">Matthew Beedham</a></li>
+                                <li>38 minutes ago</li>
+                            </ul>
+                        </div>
+
+                    </div>
+                    <div class="cards">
+                        <a class="img2" href="#"></a>
+                        <!-- <img src="./assets/images/agile-marketing-gq-540x270.png"> -->
```

Comment:
- [ ] Please remove this commented code.

Reply:
Thank you for such a detailed review. I really wasn't expecting such a detailed inspection. This issue is fixed!